packages/core/src/sass/globals.scss

```diff
@@ -47,7 +47,7 @@ $uikit-maxwidth: 42em; //Answer to life, the universe, and everything (keeps thi
 $uikit-colour-Hero: #115361 !global; //color variables (yes! the actual spelling doesn't have a u)
```

Comment:
http://sass-lang.com/guide#topic-4

You are very right here ;) We will change them all into partials.